spec/stylesheet.rb

```diff
@@ -249,7 +249,7 @@
       view.origin.x.should == 1
       view.frame = [[5,0],[5,5]]
       view.origin.x.should == 5
-      q.all.reapply_styles
+      q.reapply_styles
```

Comment:
so the change here is because the test failed after my changes to `.all` when in a sub-selection.

Because there are no children under the UIView we appended, we get no results on `all` now (previously `all` would get all views from the entire context, and thus still restyle this)

does this make sense?